packages/storage-web/src/index.ts

```diff
@@ -5,3 +5,94 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
+import {
+  IStorageBackend,
+  EmptyStorageModel,
+  StorageOptions,
+} from '@react-native-community/async-storage';
+
+class WebStorage<T extends EmptyStorageModel = EmptyStorageModel>
+  implements IStorageBackend<T> {
+  storage: any;
+  constructor(sessionStorage: boolean | string = false) {
+    this.storage = sessionStorage ? window.sessionStorage : window.localStorage;
+  }
+
+  async getSingle<K extends keyof T>(
+    key: K,
+    opts?: StorageOptions,
+  ): Promise<T[K] | null> {
+    if (opts) {
+      // noop
+    }
+    return this.storage.getItem(key);
+  }
+
+  async setSingle<K extends keyof T>(
+    key: K,
+    value: T[K],
+    opts?: StorageOptions,
+  ): Promise<void> {
+    if (opts) {
+      // noop
+    }
+    return this.storage.setItem(key, value);
+  }
+
+  async getMany<K extends keyof T>(
+    keys: Array<K>,
+    opts?: StorageOptions,
+  ): Promise<{[k in K]: T[k] | null}> {
+    if (opts) {
+      // noop
+    }
+    return Promise.all(keys.map(k => this.storage.getItem(k)));
```

Comment:
`getMany` is expected to return an object, where each key is one of provided `keys`, with value corresponding to its value in storage.

So you need to iterate over all keys, read them from storage and save value.